src/Dotnet.Script.Core/Templates/omnisharp.json.template

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "script": {
-    "enableScriptNuGetReferences": true
+    "enableScriptNuGetReferences": true,
+    "defaultTargetFramework": "netcoreapp2.0"
```

Comment:
should we remove the directive from the scaffolded `main.csx`?
also, maybe this value could be passed from the scaffolder so that we don't need to manually maintain the template

Reply:
I think maybe we should wait with removing the directive until the next version of the C# extension for Visual Studio Code. If we remove the directive now we will break intellisense

Reply:
When the next release of OmniSharp is shipped via the C# extension we can change this. We can even do the *nix shebang when running on *nix. But until that happens, I think it would be best to leave it as it is